cs_upload.py

```diff
@@ -74,15 +74,41 @@ class GraphQLClient:
     api_url: str
     ca_cert: Optional[str]
 
-    def __init__(self, api_key: str, api_url: str, ca_cert: Optional[str]):
+    def __init__(
+        self,
+        api_key: str,
+        api_url: str,
+        ca_cert: Optional[str],
+        check_ca_cert: bool = True,
+    ):
         self.api_key = api_key
         self.api_url = api_url
         self.ca_cert = ca_cert
+        self.check_ca_cert = check_ca_cert
 
     def _post(self, data: str) -> CompletedProcess:
         query = ["curl", "--request", "POST"]
-        if self.ca_cert:
+        if self.ca_cert and self.check_ca_cert:
+            query += ["--cacert", f"{self.ca_cert}"]
+        if not self.check_ca_cert:
+            query += ["--insecure"]
+        query += [
+            "--header",
+            f"API-KEY: {self.api_key}",
+            "--header",
+            "Content-Type: application/json",
+            "--data",
+            data,
+            self.api_url,
+        ]
+        return run(query, capture_output=True)
+
+    def _put(self, data: str) -> CompletedProcess:
```

Comment:
**suggestion**: Remove this method. Do we really need to use `PUT` for GraphQL queries?